dashboard/src/components/OperatorNew/OperatorNew.v2.tsx

```diff
@@ -270,7 +270,7 @@ export default function OperatorNew({ namespace, operatorName, cluster }: IOpera
               </div>
               <div className="clr-form-control">
                 <CdsButton type="submit" disabled={disableInstall}>
-                  Submit
+                  Deploy
```

Comment:
another interesting thing is that `expect(page).toClick("cds-button", { text: "Deploy" });` works but if the text is `Submit` it doesn't. Again, who knows why...

Reply:
That is both odd at first, but actually helpful. My guess: it doesn't work with "Submit" because there are multiple `cds-button`'s with the text "Submit". If you look at a page and inspect for `cds-button` you'll see there's at least the create namespace form on every page which has this, and is the first one to be found. So the scripts were probably clicking that one :)